### DISCUSSION TITLE
usb_msc_wireless_disk for this example in esp-iot-solution are you still working on

### DISCUSSION THREAD

**Discussion by truufseir** [espressif/esp-iot-solution#444] [General]
Are you still working on this demo. When will it be finished. Can i contribute to the development. I am having a slite problem when i upload a file after changing the flash size to 10mb it shows up in the web interface but not in the mass storage drive, Why do i need to remount and can you fix this bug so that it shows up ty

**Comment by leeebo** [comment]
@truufseir Hi, please check this https://github.com/espressif/esp-iot-solution/issues/301
